packages/web/src/components/atoms/Svgs/tsx/StakesSocial.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
```

Comment:
The Svg components are automatically generated from SVG files by [this command](https://github.com/dev-protocol/stakes.social/blob/master/packages/web/package.json#L11). I just ran this command in my local environment and there is a difference in the file, did you run this command?

Reply:
Yeah, I ran the command. But I didn't like the fact that the `width` and `height` attribute is set by the default, so I removed it.

I passing `undefined` when I want to ignore the default value because I think it's better if the deliverables are identical.

Reply:
...and, I wanted to change the stroke and fill colors by the property(the `themeColor`).

Reply:
I understand and thank you for fixing the problem by fixing the SVG.